images/win/Windows2022-Readme.md

```diff
@@ -55,18 +54,18 @@
 - azcopy 10.21.0
 - Bazel 6.3.2
 - Bazelisk 1.18.0
-- Bicep 0.21.1
+- Bicep 0.22.6
 - Cabal 3.10.1.0
-- CMake 3.27.6
-- CodeQL Action Bundles 2.14.5 2.14.6
+- CMake 3.26.4
```

Comment:
Yep, it broke us :slightly_frowning_face: 

https://github.com/actions/runner-images/issues/8598